supybot_fedora/plugin.py

```diff
@@ -311,8 +335,7 @@ def yield_github_repos(self, username):
 
     def yield_github_pulls(self, username, repo):
         self.log.info("Finding github pull requests for %r %r" % (username, repo))
-        tmpl = "https://api.github.com/repos/{username}/{repo}/" + \
-            "pulls?per_page=100"
+        tmpl = "https://api.github.com/repos/{username}/{repo}/" + "pulls?per_page=100"
```

Comment:
You could merge those two strings, that's probably a black artefact.

Reply:
thanks! merged these together